prometheus_es_exporter/__init__.py

```diff
@@ -354,11 +360,18 @@ def main():
         level=logging.DEBUG if args.verbose else log_level
     )
     logging.captureWarnings(True)
-
     port = args.port
     es_cluster = args.es_cluster.split(',')
 
-    if args.ca_certs:
+    if args.client_cert and args.client_key and args.ca_certs:
```

Comment:
Can we just set `client_cert` and `client_key` in the `Elasticsearch()` construction in the existing `if args.ca_certs:` case, and if their command line arguments aren't set, they'll just be `None`/disabled? Saves adding an extra case.